packages/pigeon/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 * [swift] Adds support for platform checks of imports of ProxyApis.
 * Updates minimum supported SDK version to Flutter 3.22/Dart 3.4.
+* [dart] fixes [160501](https://github.com/flutter/flutter/issues/160501).
```

Comment:
This should be it's own version update 22.7.3

I think it should read `Adds support for empty data classes`
This shouldn't be added to dart if it isn't supported on all platforms also, so no [dart] needed

Reply:
Done

Also added generating `@Suppress("UNUSED_PARAMETER")` for `fromList()` method of empty classes (Kotlin) as tests were failing due to linter

However I am facing issues at some CI steps:

```
Infra Failure: Step('Run package tests.reset XCode') (canceled) (retcode: None)
Task timed out.
```